ceph.spec.in

```diff
@@ -1432,6 +1434,11 @@ fi
 if [ $1 -eq 1 ] ; then
 /usr/bin/systemctl start ceph-osd.target >/dev/null 2>&1 || :
 fi
+%if 0%{?sysctl_apply}
+    %sysctl_apply 90-ceph-osd.conf
+%else
+    /usr/lib/systemd/systemed-sysctl %{_sysctldir}/90-ceph-osd.conf > /dev/null 2>&1 || :
+%endif
```

Comment:
here we have "90-ceph-osd.conf" but elsewhere in the same PR it is "30-ceph-osd.conf" - why the difference?